lgc/patch/PatchSetupTargetFeatures.cpp

```diff
@@ -162,6 +162,15 @@ void PatchSetupTargetFeatures::setupTargetFeatures(Module *module) {
       targetFeatures += ",+cumode";
     }
 
+#if LLVM_MAIN_REVISION && LLVM_MAIN_REVISION < 414671
+    // Old version of the code
+#else
+    // New version of the code (also handles unknown version, which we treat as latest)
+    // Enable flat scratch for gfx10.3+
+    if (gfxIp.major == 10 && gfxIp.minor >= 3)
```

Comment:
`gfxIp >= GfxIpVersion({10, 3})` is a bit more readable and a bit more future-proof.